--- Makefile.global	2012-06-26 14:48:24.000000000 -0400
+++ Makefile.new	2012-06-26 14:51:37.000000000 -0400
@@ -201,7 +201,7 @@
 
 # Compilers
 
-CPP = gcc -E
+CPP = gcc-4.2 -E
 CPPFLAGS =  -I/usr/include/libxml2  -I/usr/local/include/libxml2 -I/usr/local/include
 
 ifdef PGXS
@@ -213,7 +213,7 @@
 endif
 endif # not PGXS
 
-CC = gcc
+CC = gcc-4.2
 GCC = yes
 SUN_STUDIO_CC = no
 CFLAGS = -isysroot /Developer/SDKs/MacOSX10.5.sdk -mmacosx-version-min=10.5 -headerpad_max_install_names -arch i386 -arch x86_64 -Wall -Wmissing-prototypes -Wpointer-arith -Wdeclaration-after-statement -Wendif-labels -Wformat-security -fno-strict-aliasing -fwrapv